BootloaderCommonPkg/Library/IppCryptoLib/auth/pcpngrsaencodec.c

```diff
@@ -169,16 +169,14 @@ void gsRSApub_cipher(IppsBigNumState* pY,
 
    BNU_CHUNK_T* dataY = BN_NUMBER(pY);
 
-   if (m != NULL) {
-      cpSize nsY = m->publicExpFun(dataY,
-                                  BN_NUMBER(pX), BN_SIZE(pX),
-                                  RSA_PUB_KEY_E(pKey), BITS_BNU_CHUNK(RSA_PUB_KEY_BITSIZE_E(pKey)),
-                                  RSA_PUB_KEY_NMONT(pKey),
-                                  pBuffer);
-      FIX_BNU(dataY, nsY);
-      BN_SIZE(pY) = nsY;
-      BN_SIGN(pY) = ippBigNumPOS;
-   }
+   cpSize nsY = m->publicExpFun(dataY,
```

Comment:
KlocWork reports an error because of unnecessary NULL check?

Reply:
No.  It reported m->praviteExpFun could be NULL.  
But the previous fix added  check to verify if m is NULL or not for many other cases.  That fix does not really address the issue reported by KW and it is not necessary.  m itself is not a problem and it won't be NULL in the code flow.    So remove the "m" check here to match the original code.

Reply:
Got it. Thanks for your explanation.